modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/function/MBFunction_toBool.java

```diff
@@ -0,0 +1,66 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2018, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.mbstyle.function;
+
+import org.geotools.filter.FunctionExpressionImpl;
+import org.geotools.filter.capability.FunctionNameImpl;
+import org.opengis.filter.capability.FunctionName;
+
+/**
+ * Take an object as an argument and returns a boolean value.
+ * The result is false when then input is an empty string, 0, false, null,
+ * or NaN; otherwise it is true.
+ *
+ * This function is a helper to accommodate MBType expressions. Expressions in this section are provided for the purpose of
+ * testing for and converting between different data types like strings, numbers, and boolean values.
+ */
+class MBFunction_toBool extends FunctionExpressionImpl {
+
+    public static FunctionName NAME = new FunctionNameImpl("mbToBool");
```

Comment:
would prefer making a naming convention here if we really need to introduce casting: toBoolean, toInteger, toFloat, etc....